apps/storybook/src/forms/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { ToggleGroup, ToggleGroupItem } from "@optiaxiom/react/unstable";
+
+export default {
+  argTypes: {
+    onClick: { action: "click" },
+  },
+  component: ToggleGroup,
+} as Meta<typeof ToggleGroup>;
+
+type Story = StoryObj<typeof ToggleGroup>;
+
+export const Basic: Story = {
+  render: () => {
+    return (
+      <ToggleGroup>
+        <ToggleGroupItem value="a">A</ToggleGroupItem>
+        <ToggleGroupItem value="b">B</ToggleGroupItem>
+        <ToggleGroupItem value="c">C</ToggleGroupItem>
+      </ToggleGroup>
+    );
+  },
```

Comment:
let's move the children to the `default` args and avoid using `render` in the stories